packages/synapse-react-client/src/synapse-queries/auth/useTwoFactorEnrollment.ts

```diff
@@ -15,7 +15,7 @@ import SynapseClient from '../../synapse-client'
 import { useSynapseContext } from '../../utils/context/SynapseContext'
 
 export function useStartTwoFactorEnrollment(
-  options?: UseMutationOptions<TotpSecret, SynapseClientError>,
+  options?: Partial<UseMutationOptions<TotpSecret, SynapseClientError>>,
```

Comment:
Since `UseQueryOptions` and `UseMutationOptions` now includes the `queryFn`/`mutationFn` as a required argument, our option override types must now be wrapped in `Partial`, since the `queryFn` is already hard-coded